openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -32,20 +36,24 @@ def course_home_url(course):
     )
 
 
-class TestCourseHomePage(SharedModuleStoreTestCase):
+class CourseHomePageTestCase(SharedModuleStoreTestCase):
```

Comment:
It seems we don't have unit tests at all for these features, but they kind of got tested through the course home page. I've made a one point story to add these tests:

https://openedx.atlassian.net/browse/LEARNER-1780